scripts/install.sh

```diff
@@ -65,40 +67,51 @@ curl -sL "https://api.github.com/repos/${GITHUB_REPO}/releases/latest" \
   | grep -oE '[^"]*$'
 }
 
+get_latest_nightly_release() {
+  curl -sL "https://api.github.com/repos/${GITHUB_REPO}/releases"  \
+  | grep -oE '"tag_name": "[^"]*' \
+  | grep -oE '[^"]*$' \
+  | head -1
+}
+
 download_version() {
-version="${1}"
-sudo=""
+  version="${1}"
+  sudo=""
 
-if [ ! -w "${TARGET_DIR}" ]; then
-  if [ -x "$(command -v 'sudo')" ]; then
-    echo "Target diectory (${TARGET_DIR}) is not writable, trying to use sudo"
-    sudo="sudo"
+  if [ ! -w "${TARGET_DIR}" ]; then
+    if [ -x "$(command -v 'sudo')" ]; then
+      echo "Target diectory (${TARGET_DIR}) is not writable, trying to use sudo"
+      sudo="sudo"
+    fi
+    ${sudo} [ -w "${TARGET_DIR}" ] \
+      || fail "Target diectory (${TARGET_DIR}) is not writable (destination can be changed using \$TARGET_DIR variable)"
   fi
-  ${sudo} [ -w "${TARGET_DIR}" ] \
-    || fail "Target diectory (${TARGET_DIR}) is not writable (destination can be changed using \$TARGET_DIR variable)"
-fi
 
-curl -L -o- "https://github.com/${GITHUB_REPO}/releases/download/${version}/${APP_NAME}-${version}-${TARGET_OS}-${TARGET_ARCH}.tar.gz" \
-  | ${sudo} tar -xz -C "${TARGET_DIR}"
+  curl -L -o- "https://github.com/${GITHUB_REPO}/releases/download/${version}/${APP_NAME}-${version}-${TARGET_OS}-${TARGET_ARCH}.tar.gz" \
+    | ${sudo} tar -xz -C "${TARGET_DIR}"
 }
 
 check_colors
 check_binaries
 echo ">>> ${GREEN}${APP_NAME} installation script${NOCOL} <<<"
 
-while getopts "hd:a:o:" OPTION
+while getopts "hd:a:o:n:" OPTION
 do
      case $OPTION in
          h) usage; exit;;
          d) TARGET_DIR="$OPTARG";;
          a) TARGET_ARCH="$OPTARG";;
          o) TARGET_OS="$OPTARG";;
+         n) TARGET_NIGHTLY="$OPTARG";;
          ?) usage; exit;;
      esac
 done
 
 echo "${YELLOW}>${NOCOL} Detecting latest version"
 release="$(get_latest_release)"
+if [ ${TARGET_NIGHTLY} -eq 1 ]; then
```

Comment:
Can we do if/else here? Otherwise, we're always downloading the latest tagged release info, even if that is not required.

Reply:
Also
```
In scripts/install.sh line 112:
if [ ${TARGET_NIGHTLY} -eq 1 ]; then
     ^---------------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
yep will fix